hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/precompile/impl/TokenCreatePrecompile.java

```diff
@@ -822,6 +935,29 @@ public static TokenCreateWrapper decodeFungibleCreateWithFeesV2(
         return tokenCreateWrapper;
     }
 
+    public static TokenCreateWrapper decodeFungibleCreateWithFeesV3(
```

Comment:
`decodeFungibleCreateWithFees`, `decodeFungibleCreateWithFeesV2` and `decodeFungibleCreateWithFeesV3` share almost a common body. It can be extracted in a private method. The only difference seems to be the `Tuple` `decodeFunctionCall`.